### DISCUSSION TITLE
Welcome to vali-admin Discussions!

### DISCUSSION THREAD

**Discussion by pratikborsadiya** [pratikborsadiya/vali-admin#128] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

**Comment by sowailem** [comment]
Thank you Pratik and all members.

All the best,,,
